apteryx.c

```diff
@@ -1129,8 +1168,8 @@ _set_multi (GNode *node, gpointer data)
     return FALSE;
 }
 
-bool
-apteryx_set_tree_full (GNode* root, uint64_t ts, bool wait_for_completion)
+static bool
+_apteryx_set_tree_full (GNode* root, uint64_t ts, uint64_t id, set_flags flags)
```

Comment:
I think we can just change the signature for apteryx_set_tree_full to take flags - no one is using it at the moment.